aptos-move/aptos-vm-types/src/output.rs

```diff
@@ -78,7 +91,7 @@ impl VMOutput {
         }
 
         // Try to materialize deltas and add them to the write set.
-        let (change_set, gas_used, status) = self.unpack();
+        let (change_set, gas_used, status) = self.unpack_with_fee_statement();
```

Comment:
nit: gas_used --> fee_statement

Reply:
Fixed in here: https://github.com/aptos-labs/aptos-core/pull/8694/files#diff-f0dbd4cd6470ed938cbaefe5e9eafe41ef5f79398db858fff90388586646fce3R94